mix.exs

```diff
@@ -3,30 +3,26 @@ defmodule Exnumerator.Mixfile do
 
   def project do
     [
-      app:             :exnumerator,
-      version:         "1.6.1",
-      elixir:          "~> 1.5",
-      build_embedded:  Mix.env == :prod,
-      start_permanent: Mix.env == :prod,
-      description:     description(),
-      package:         package(),
-      deps:            deps()
+      app: :exnumerator,
+      version: "1.6.1",
+      elixir: "~> 1.5",
+      build_embedded: Mix.env() == :prod,
+      start_permanent: Mix.env() == :prod,
+      description: description(),
+      package: package(),
+      deps: deps()
     ]
   end
 
   def application do
     [
-      applications: apps(),
+      applications: []
     ]
   end
 
-  defp apps do
-    []
-  end
-
   defp deps do
     [
-      {:ex_doc, "~> 0.16", only: :dev, runtime: false},
+      {:ex_doc, "~> 0.16", only: :dev, runtime: false}
```

Comment:
Can you enable comma for the last line?

Reply:
@KamilLelonek I wouldn't, `mix format` did it job. Roll with whatever is the default for the community.